sbt-coursier/src/sbt-test/sbt-coursier/classifiers/build.sbt

```diff
@@ -0,0 +1,18 @@
+scalaVersion := "2.11.8"
+
+libraryDependencies += ("org.jclouds.api" % "nova" % "1.5.9")
+  .classifier("tests")
+
+classpathTypes += "test-jars"
+
+resolvers ++= Seq(
+  MavenRepository("Maven Central", "http://central.maven.org/maven2/")
```

Comment:
I switched repos to ```MavenRepository("Repo1", Resolver.DefaultMavenRepositoryRoot)```

that made no difference.  I'm not seeing the artifact when I do ```show managedClasspath```